go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/coredns/coredns
 
-go 1.13
+go 1.16
```

Comment:
This is a "minimum version" spec. This should only be updated if there is code that won't compile on older versions of Go.